build.gradle

```diff
@@ -0,0 +1,25 @@
+plugins {
+	id 'org.springframework.boot' version '2.4.2'
+	id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+	id 'java'
+}
+
+group = 'wolox'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	compile('org.springframework.boot:spring-boot-starter-data-jpa')
+	compile('org.springframework.boot:spring-boot-starter-web')
+	compile group: 'org.postgresql', name: 'postgresql', version: '42.2.5'
```

Comment:
David, it's just a suggestion.
The dependencies can be uniform, that is:
- compile group: 'org.postgresql', name: 'postgresql', version: '42 .2.5 '-> compile('org.postgresql: postgresql: 42.2.5')